op-node/rollup/derive/attributes.go

```diff
@@ -121,9 +121,19 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 		return nil, NewCriticalError(fmt.Errorf("failed to create l1InfoTx: %w", err))
 	}
 
-	txs := make([]hexutil.Bytes, 0, 1+len(depositTxs)+len(upgradeTxs))
+	var postDeposits []hexutil.Bytes
+	if ba.rollupCfg.IsInterop(nextL2Time) {
+		depositsCompleteTx, err := DepositsCompleteBytes(ba.rollupCfg, sysConfig, seqNumber, l1Info, nextL2Time)
```

Comment:
The `DepositsCompleteBytes` function should be able to accept less arguments, its stateless just returns a binary serialized deposit tx where `tx.to = predeploys.L1Block`, `tx.from = depositor account` and `tx.data = keccak256("setL1BlockIsthmus()")[0:4]`. The source hash would also need to be handled properly, this is where the arguments may come into play, the source hash is described in the specs and also can be followed from looking at how other L1 block deposit txs are generated